config/dev.webpack.config.js

```diff
@@ -2,13 +2,14 @@
 const { resolve } = require('path');
 const BundleAnalyzerPlugin = require('webpack-bundle-analyzer').BundleAnalyzerPlugin;
 const config = require('@redhat-cloud-services/frontend-components-config');
+const HttpsProxyAgent = require('https-proxy-agent');
```

Comment:
This proxy agent is a part of the common config. Are you sure we need it here again?

Reply:
Yup, because there's a env flag `API_ENDOINT` which deals with redirecting to custom API, however that requires `squid` proxy for prod and stage envs.